sphinx/source/docs/user_guide/tools.rst

```diff
@@ -872,7 +867,7 @@ editing the underlying ``ColumnDataSource`` data. Like other drawing
 tools, the renderers that are to be edited must be supplied explicitly
 as a list.
 
-The tool will automatically modify the columns on the data source
+The tool automatically modifies the columns on the data source
 corresponding to the ``xs`` and ``ys`` values of the glyph. Any
 additional columns in the data source will be padded with the declared
 ``empty_value``, when adding a new point. Any newly added patch or
```

Comment:
re line 879 below: another "we" to replace ("Using the ``num_objects`` property, we can ensure that once")